File: Entities/DTOs/CarCreateDto.cs
using Core.Entities;
using System;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Entities.DTOs
{
    public class CarCreateDto : IDto
    {

        public Guid BrandId { get; set; }
        public Guid ColorId { get; set; }
        public Guid CarTypeId { get; set; }
        public Guid FuelTypeId { get; set; }
        public Guid GearTypeId { get; set; }
        public Guid InsertedUserId { get; set; }
        public string CarName { get; set; }
        public string Model { get; set; }
        public string Description { get; set; }
        public int MinFindexScore { get; set; }

    }
}

File: ui/src/app/models/CarCreateDto.ts
import { Guid } from "guid-typescript";


export class CarCreateDto {
  carName: string;
  brandId: Guid;
  colorId: Guid;
  model: Guid;
  carTypeId: Guid;
  fuelTypeId: Guid;
  gearTypeId: Guid;
  insertedUserId: Guid;
  description: string;
  minFindexScore: number;
}
